integration_tests/tsconfig.json

```diff
@@ -7,12 +7,6 @@
     "types": ["cypress", "cypress-axe", "express", "express-session"],
     "esModuleInterop": true,
     "skipLibCheck": true,
-    "typeRoots": [
```

Comment:
Ah so according to https://www.typescriptlang.org/tsconfig#typeRoots:

> By default all visible ”@types” packages are included in your compilation. Packages in node_modules/@types of any enclosing folder are considered visible. For example, that means packages within ./node_modules/@types/, ../node_modules/@types/, ../../node_modules/@types/, and so on.

I'm not sure of the benefit of including them previously, maybe there was a package we (or someone else working on one of the MOJ TypeScript projects) had to explicitly specify where the types were coming from? Sadly there's not a lot of context as to why this changed [here](https://github.com/ministryofjustice/hmpps-template-typescript/pull/196/commits/d4d63696ca07fb794b935fcfffd41f3957bfc9c9) in the template when updating dependencies.

Reply:
Yeah I saw that bit of documentation the other day but likewise couldn't figure out how it helped us

I think that commit just removes a comma from the `typeRoots` line. It's this [earlier commit](https://github.com/ministryofjustice/hmpps-template-typescript/commit/ccc0c0372ac78496306907a5e7cb1845b1cbe937#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0) (also dependencies) that introduces it. I wonder if it's so that all the `./server/@types` are loaded, while we set up aliases for those paths and import by those aliases where needed, so we don't need (also) to load everything everywhere?

Here's the unsquashed commit from the template: https://github.com/ministryofjustice/hmpps-template-typescript/pull/8/commits/446ae5a9a508e0a5ce80bfa8201dea49e0664a65#r533829584. Doesn't really help 🤷🏻‍♀️

Reply:
Oh I misread the commit you linked before - I was looking at the top-level tsconfig.json instead of the one in integration_tests